src/3box.js

```diff
@@ -18,6 +18,7 @@ const LevelStore = require('datastore-level')
 const didJWT = require('did-jwt')
 
 const SharedCache = require('3box-shared-cache')
+const { iframe_cache_url } = require('./config') // eslint-disable-line
```

Comment:
You don't need to import this since it's already imported on line 14 :)

Reply:
Thanks for the catch